### PR TITLE
Update: Google Cloud Run details

### PR DESCRIPTION
**Pull request by MimiDumpling**
Google Cloud Run integration needed a few details regarding integration type and permissions.

### CODE REVIEW

File: src/docs/product/integrations/gcp-cloud-run/index.mdx [hunk 1]
```diff
@@ -6,8 +6,16 @@ description: "Learn more about Sentry's Google Cloud Run integration and how it
 
 Cloud Run helps developers save time in building and deploying their applications. Sentry helps save time in resolving production issues by providing detailed debugging capabilities.
 
+This integration needs to be set up in each project for which you wish to use it. It is maintained and supported by the [Sentry community](https://forum.sentry.io/).
```
Comment (MimiDumpling) [inline src/docs/product/integrations/gcp-cloud-run/index.mdx:9]: @ajjindal will you confirm if this project level integration is maintained by the Sentry community?

File: src/docs/product/integrations/gcp-cloud-run/index.mdx [hunk 2]
```diff
@@ -6,8 +6,16 @@ description: "Learn more about Sentry's Google Cloud Run integration and how it
...
+
 ## Install and Configure
 
+<Note>
+
+Sentry owner or manager permissions permissions are required to install this integration.
```
Comment (MimiDumpling) [inline src/docs/product/integrations/gcp-cloud-run/index.mdx:15]: @ajjindal will you confirm the permission level for this integration?

File: src/docs/product/integrations/gcp-cloud-run/index.mdx (hunk 1 above)
Comment (ajjindal) [reply src/docs/product/integrations/gcp-cloud-run/index.mdx:9]: The Google Cloud Run integrations page is more of a place holder for us to be able to partner with GCP. So, we should skip this addition.

File: src/docs/product/integrations/gcp-cloud-run/index.mdx (hunk 2 above)
Comment (ajjindal) [reply src/docs/product/integrations/gcp-cloud-run/index.mdx:15]: The Google Cloud Run integrations page is more of a place holder for us to be able to partner with GCP. So, we should skip this addition.

**Comment by PeloWriter** [comment]
These updates aren't applicable to Cloud Run.

**Pull request closed without merging**